src/main/java/pl/touk/sputnik/processor/spotbugs/SpotBugsProcessor.java

```diff
@@ -121,4 +130,23 @@ private String getExcludeFilterFilename() {
         log.info("Using SpotBugs exclude filter file {}", excludeFilterFilename);
         return excludeFilterFilename;
     }
+
+    private void loadAllSpotbugsPlugins() {
+        String pluginsLocation = config.getProperty(GeneralOption.SPOTBUGS_PLUGINS_LOCATION);
+        if (pluginsLocation != null) {
+            File[] pluginsList = new File(pluginsLocation).listFiles();
+            for (File plugin : Objects.requireNonNull(pluginsList)) {
+                if (plugin.getName().contains(".jar")) {
+                    log.info("SpotBugs additional plugin loading: file://{}", plugin);
+                    try {
+                        Plugin.getAllPlugins().add(Plugin.addCustomPlugin(new URI("file://" + plugin.getAbsoluteFile())));
```

Comment:
This is strange. Is `Plugin` a singleton for FindBugs?

Reply:
Ok, acctually I've found out that addCustomPlugin was enough for adding plugin. It is already fixed in the latest commits.